package.json

```diff
@@ -15,8 +15,10 @@
     "react": "^16.13.1",
     "react-dom": "^16.13.1",
     "react-plotly.js": "^2.4.0",
+    "react-redux": "^7.2.6",
```

Comment:
Looks great. Just a reminder, we may also need redux-thunk to make async api calls.

Reply:
Do we also want to add a development dependency to get request logging? Like redux logger?